app/renderer/ui/containers/WalletForm.tsx

```diff
@@ -0,0 +1,364 @@
+                break
+              case newMnemonicsErrors.INVALID_MNEMONICS_TYPE.value:
+                this.setState({ mnemonicsErrorMessage: "Error generating mnemonics" })
+                break
+              default:
+                assertNever(mnemonicsResponse.error)
+            }
+          }
+        }
+        this.to(WALLET_SEED_BACKUP)()
+      }
+    }
+
+    return (
+      <WalletSeedTypeSelection
+        nextStep={steps}
+      />
+    )
+  }
+
+  /**
+   * method to render seed backup step
+   *
+   * @private
+   * @memberof WalletFormContainer
+   */
+  private renderSeedBackup = () => {
+    return (
+      <WalletSeedBackup
+        mnemonics={this.state.mnemonics}
+        nextStep={this.to(WALLET_SEED_CONFIRMATION)}
+        previousStep={this.to(WALLET_SEED_TYPE_SELECTION)}
+      />
+    )
+  }
+
+  /**
+   * method to render seed confirmation step
+   *
+   * @private
+   * @memberof WalletFormContainer
+   */
+  private renderSeedConfirmation = () => {
+    const onChangeInput = async (ev: any) => {
+      this.setState({ confirmMnemonics: ev.target.value })
+      if (this.state.mnemonics === this.state.confirmMnemonics) {
+        this.setState({ mnemonicsErrorMessage: "" })
+      }
+    }
+
+    const nextStep = async () => {
+      if (this.state.mnemonics === this.state.confirmMnemonics) {
+        this.setState({ spinner: true })
+        const mnemonicsValidationResponse = await validateMnemonics(
+          this.props.services.apiClient,
+          this.state.confirmMnemonics
+        )
+        this.setState({ spinner: false })
+        if (mnemonicsValidationResponse.kind === "SUCCESS") {
+          this.to(WALLET_ENCRYPTION_PASSWORD)()
+        } else {
+          // TODO: improve error handler issue #321
+          switch (mnemonicsValidationResponse.error) {
+            case validteMnemonicsErrors.INVALID_METHOD_PARAMS.value:
+            case validteMnemonicsErrors.INVALID_XPRV.value:
+            case validteMnemonicsErrors.INVALID_MNEMONICS.value:
+            case validteMnemonicsErrors.MISMATCHING_MNEMONICS.value:
+            case validteMnemonicsErrors.UNCONSOLIDATED_UPDATE_FAILURE.value:
+              this.setState({ mnemonicsErrorMessage: "Invalid mnemonics" })
+              break
+            default:
+              assertNever(mnemonicsValidationResponse.error)
+          }
+        }
+      } else {
+        this.setState({ mnemonicsErrorMessage: "Invalid mnemonics" })
+      }
+    }
+
+    return (
+      <WalletSeedConfirmation
+        errorMessage={this.state.mnemonicsErrorMessage}
+        inputValue={this.state.confirmMnemonics}
+        onChangeInput={onChangeInput}
+        nextStep={nextStep}
+        previousStep={this.to(WALLET_SEED_BACKUP)}
+      />
+    )
+  }
+
+  /**
+   * method to render encryption password step
+   *
+   * @private
+   * @memberof WalletFormContainer
+   */
+  private renderEncryptionPassword = () => {
+    const walletEncryption = async () => {
```

Comment:
Constant functions shall never be declared inside renderer functions for performance reasons.